vendors/swedbank-pay-sdk-php/tests/VippsPaymentTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+use SwedbankPay\Api\Service\Payment\Resource\Collection\PricesCollection;
+use SwedbankPay\Api\Service\Payment\Resource\Collection\Item\PriceItem;
+use SwedbankPay\Api\Service\Payment\Transaction\Resource\Request\TransactionObject;
+use SwedbankPay\Api\Service\Vipps\Request\Purchase;
+use SwedbankPay\Api\Service\Vipps\Resource\Request\PaymentPayeeInfo;
+use SwedbankPay\Api\Service\Vipps\Resource\Request\PaymentPrefillInfo;
+use SwedbankPay\Api\Service\Vipps\Resource\Request\PaymentUrl;
+use SwedbankPay\Api\Service\Vipps\Resource\Request\Payment;
+use SwedbankPay\Api\Service\Vipps\Resource\Request\VippsPaymentObject;
+
+use SwedbankPay\Api\Service\Data\ResponseInterface as ResponseServiceInterface;
+use SwedbankPay\Api\Service\Vipps\Transaction\Request\CreateAuthorization;
+use SwedbankPay\Api\Service\Vipps\Transaction\Resource\Request\TransactionAuthorization;
+use SwedbankPay\Api\Service\Resource\Data\ResponseInterface as ResponseResourceInterface;
+
+class VippsPaymentTest extends TestCase
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class VippsPaymentTest has a coupling between objects value of 14. Consider to reduce the number of dependencies under 13.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)